pkg/kubelet/kubelet_pods.go

```diff
@@ -1484,6 +1484,10 @@ func (kl *Kubelet) determinePodResizeStatus(pod *v1.Pod, podStatus *v1.PodStatus
 // internal pod status. This method should only be called from within sync*Pod methods.
 func (kl *Kubelet) generateAPIPodStatus(pod *v1.Pod, podStatus *kubecontainer.PodStatus) v1.PodStatus {
```

Comment:
Tests are failing because I need to refactor this method to use kubecontainer.PodStatus... It requires a lot of mental load and some drawings. I will do my best to fix that tomorrow or during the weekend.